doc/osh.1

```diff
@@ -1,42 +1,85 @@
-.TH OSH "1" "November 2017" "OSH version 0.2.0" "User Commands"
-.SH NAME
-osh \- A bash-compatible UNIX shell
-.SH SYNOPSIS
-.B osh
-[\fI\,options\/\fR] \fI\,script \/\fR[\fI\,args\/\fR]
-.br
-.B osh
-[\fI\,options\/\fR] -c [\fI\,command_string\/\fR] [\fI\,args\/\fR]
-.SH DESCRIPTION
-.PP
-OSH is a command-line shell. It behaves like busybox and `osh` is a symlink to it.
-.PP
-.SH OPTIONS
-OSH accepts POSIX \fBsh\fR flags, with the following differences:
-.TP
-\fB\-n\fR
-only validate the syntax.  Also prints the AST.
-.TP
-\fB\-\-show\-ast\fR
-print the AST in addition to executing.
-.TP
-\fB\-\-ast\-format\fR
-what format the AST should be in
-.SH AUTHOR
-Written by Andy Chu.
-.SH REPORTING BUGS
-Please report bugs to <https://github.com/oilshell/oil/issues>
-.SH COPYRIGHT
-Copyright © 2016 Andy Chu. All rights reserved.
-.br
-Licensed under the Apache License, Version 2.0 (the "License");
-.br
-You may not use this file except in compliance with the License. You may obtain a copy of the License at
-.IP
-http://www.apache.org/licenses/LICENSE-2.0
-.SH SEE ALSO
-Developer documentation for
-.B OSH
-may be found online at
-.IP
-http://www.oilshell.org/release/0.2.0/doc/osh-quick-ref.html
+.\" Copyright © 2016 Andy Chu. All rights reserved.
+.\"
+.\" Licensed under the Apache License, Version 2.0 (the "License");
+.\" You may not use this file except in compliance with the License.
+.\"
+.\" You may obtain a copy of the License at:
+.\" http://www.apache.org/licenses/LICENSE-2.0
+.Dd June 28, 2019
+.Dt OSH 1
+.Os
+.Sh NAME
+.Nm osh
+.Nd command language interpreter
+.Sh SYNOPSIS
+.Nm
+.Op Ar options
+.Ar script
+.Op Ar args
+.Nm
+.Op Ar options
+.Fl c
+.Op command_string
+.Op args
+.Sh DESCRIPTION
+The
+.Nm
+utility is a
+.Xr bash 1 Ns -compatible
+command language intepreter:
+it reads commands either from standard input, a
+.Ar script
+file or command-line arguments and executes them.
+.Pp
+.Nm
+accepts POSIX
+.Xr sh 1
+arguments, with the following differences:
+.Bl -tag -width Ds
+.It Fl n
+Only validate the syntax.
+Also prints the AST.
+.It Fl -show-ast
+Print the AST in addition to executing.
+.It Fl -ast-format
+What format the AST should be in.
+.El
+.Sh ENVIRONMENT
+OSH_HIJACK_SHEBANG
+OSH_CRASH_DUMP_DIR
+.Sh FILES
```

Comment:
This could be rewritten as:

```mdoc
.Sh FILES
.Bl -tag -width "~/.config/oil/oshrc" -compact
.It Pa ~/.config/oil/oshrc
User configuration file.
Sourced on startup.
.El
```

`.Pa` is what is used to reference system paths/files

Reply:
updated with path link. I'm leaving the wording - yours implies that there's multiple configs, and I believe (in addition to the specific path), that multiple configs are being avoided. @andychu does that sound right?